test/integration/test-fixtures/image-pkg-coverage/pkgs/var/lib/pacman/local/pacman-6.0.1-5/files

```diff
@@ -23,383 +23,4 @@ usr/lib/libalpm.so
-usr/share/makepkg/lint_pkgbuild/pkgver.sh
-usr/share/makepkg/lint_pkgbuild/provides.sh
-usr/share/makepkg/lint_pkgbuild/source.sh
-usr/share/makepkg/lint_pkgbuild/util.sh
-usr/share/makepkg/lint_pkgbuild/variable.sh
-usr/share/makepkg/reproducible.sh
-usr/share/makepkg/reproducible/
-usr/share/makepkg/reproducible/python.sh
-usr/share/makepkg/source.sh
-usr/share/makepkg/source/
-usr/share/makepkg/source/bzr.sh
-usr/share/makepkg/source/file.sh
-usr/share/makepkg/source/fossil.sh
-usr/share/makepkg/source/git.sh
-usr/share/makepkg/source/hg.sh
-usr/share/makepkg/source/local.sh
-usr/share/makepkg/source/svn.sh
-usr/share/makepkg/srcinfo.sh
-usr/share/makepkg/tidy.sh
-usr/share/makepkg/tidy/
-usr/share/makepkg/tidy/docs.sh
-usr/share/makepkg/tidy/emptydirs.sh
-usr/share/makepkg/tidy/libtool.sh
-usr/share/makepkg/tidy/purge.sh
-usr/share/makepkg/tidy/staticlibs.sh
-usr/share/makepkg/tidy/strip.sh
-usr/share/makepkg/tidy/zipman.sh
-usr/share/makepkg/util.sh
-usr/share/makepkg/util/
-usr/share/makepkg/util/compress.sh
-usr/share/makepkg/util/config.sh
-usr/share/makepkg/util/dirsize.sh
-usr/share/makepkg/util/error.sh
-usr/share/makepkg/util/message.sh
-usr/share/makepkg/util/option.sh
-usr/share/makepkg/util/parseopts.sh
-usr/share/makepkg/util/pkgbuild.sh
-usr/share/makepkg/util/schema.sh
-usr/share/makepkg/util/source.sh
-usr/share/makepkg/util/util.sh
-usr/share/man/
-usr/share/man/man1/
-usr/share/man/man1/makepkg-template.1.gz
-usr/share/man/man3/
-usr/share/man/man3/libalpm.3.gz
-usr/share/man/man3/libalpm_cb.3.gz
-usr/share/man/man3/libalpm_databases.3.gz
-usr/share/man/man3/libalpm_depends.3.gz
-usr/share/man/man3/libalpm_errors.3.gz
-usr/share/man/man3/libalpm_files.3.gz
-usr/share/man/man3/libalpm_groups.3.gz
-usr/share/man/man3/libalpm_handle.3.gz
-usr/share/man/man3/libalpm_list.3.gz
-usr/share/man/man3/libalpm_log.3.gz
-usr/share/man/man3/libalpm_misc.3.gz
-usr/share/man/man3/libalpm_options.3.gz
-usr/share/man/man3/libalpm_packages.3.gz
-usr/share/man/man3/libalpm_sig.3.gz
-usr/share/man/man3/libalpm_trans.3.gz
-usr/share/man/man5/
-usr/share/man/man5/BUILDINFO.5.gz
-usr/share/man/man5/PKGBUILD.5.gz
-usr/share/man/man5/alpm-hooks.5.gz
-usr/share/man/man5/makepkg.conf.5.gz
-usr/share/man/man5/pacman-hooks.5.gz
-usr/share/man/man5/pacman.conf.5.gz
-usr/share/man/man8/
-usr/share/man/man8/makepkg.8.gz
-usr/share/man/man8/pacman-conf.8.gz
-usr/share/man/man8/pacman-key.8.gz
-usr/share/man/man8/pacman.8.gz
-usr/share/man/man8/repo-add.8.gz
-usr/share/man/man8/repo-remove.8.gz
-usr/share/man/man8/vercmp.8.gz
-usr/share/pacman/
-usr/share/pacman/PKGBUILD-split.proto
-usr/share/pacman/PKGBUILD-vcs.proto
-usr/share/pacman/PKGBUILD.proto
-usr/share/pacman/proto.install
-usr/share/pkgconfig/
-usr/share/pkgconfig/libmakepkg.pc
-usr/share/zsh/
-usr/share/zsh/site-functions/
-usr/share/zsh/site-functions/_pacman
-var/
-var/cache/
-var/cache/pacman/
-var/cache/pacman/pkg/
-var/lib/
-var/lib/pacman/
-
-%BACKUP%
```

Comment:
With this change we won't see if our backup files are parsed or not. Maybe we should leave them?

Reply:
Ahh you're right - I was just trying to shorten the output of the SBOM and got a little over enthusiastic. Thanks for the catch